tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.match.txt

```diff
@@ -1,2 +1,2 @@
 REGEXP_MATCH([str0], 'OFFICE')
-REGEXP_MATCH([str0], 'Z')
```

Comment:
@tabsdavis revert this line to have the `Z`

Reply:
Made the revert to the match file. That solves the expected issue. Added exclusions to mentioned ini files as well as sqlserver.ini